src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -444,8 +444,15 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
         this._handleLiveDurationChange();
       }
     };
-    if (startTime && startTime > -1) {
-      this._videoElement.currentTime = startTime;
+    const setStartTime = seekTo => {
+      if (seekTo && seekTo > -1) {
+        this._videoElement.currentTime = seekTo;
+      }
+    };
+    setStartTime(startTime);
+    if (!this.isLive()) {
+      //for stream that have two duration change - like ima dai sample
+      this._eventManager.listen(this._videoElement, Html5EventType.DURATION_CHANGE, () => setStartTime(startTime));
```

Comment:
do we understand why this is happening? this looks like an issue in the specific media tested rather than an issue with player.
What is the duration of the video after the first durationchange event? is it different from the second one? please note that DAI playlists are stitched playlists so maybe this is the issue

Reply:
OK, Yes the duration is different in milliseconds and it comes from the difference between the video and audio segments. @OrenMe does it OK that there is a difference? or an issue with the stream? I found out different IMA DAI media that doesn't have this difference and it works well. Not sure if we want it in this case.

Reply:
So yes, this is issue in stream.
I think it is ok to reject this and raise a bug with IMA with this issue, rather than adding fixes for invalid cases.
These kind of issues can also break the video ended events in some cases.

Reply:
Agree